gradle-diff-task-dependencies/plugin/src/test/groovy/com/novoda/gradle/diffdependencytask/ConditionalDependencyEvaluatorTest.groovy

```diff
@@ -0,0 +1,104 @@
+package com.novoda.gradle.diffdependencytask
+
+import org.junit.Before
+import org.junit.Test
+import org.mockito.InjectMocks
+import org.mockito.Mock
+
+import static com.google.common.truth.Truth.assertThat
+import static org.mockito.Mockito.when
+import static org.mockito.MockitoAnnotations.initMocks
+
+class ConditionalDependencyEvaluatorTest {
+
+    @Mock
+    private ConditionalDependencyRepository dependenciesRepository
+    @Mock
+    private GroovyCallable<List<String>> changedFilesProvider
+    @InjectMocks
```

Comment:
Main problem with `InjectMocks` is when you change c'tor and add extra parameter, `InjectMocks` doesn't throw an exception or notify you about missing parameter but simply injects a `null` there. 💥 💣 💥